### DISCUSSION TITLE
Submission times refresh time

### DISCUSSION THREAD

**Discussion by aptx1231** [PaddlePaddle/PaddleSpatial#193] [BaiduKDDCup2022]
About the refresh time of the number of submissions, the description says AOE time 23:59 refresh, corresponding to Beijing time is 19:59, but now and 21:00 more in Beijing, why not refresh it?

![image](https://user-images.githubusercontent.com/35984903/174440019-42b01610-c773-41ff-9ee0-1fd5bb2b6196.png)

Is it because our last submission was before 23:59 AOE time, but the review results came out after 23:59 AOE time, resulting in a BUG?

**Comment by zhoujingbo** [comment]
Is it refreshed now? 

" our last submission was before 23:59 AOE time, but the review results came out after 23:59 AOE time", I think  it is the reason.

**Reply by aptx1231** [reply]
These days the submission times are refreshed after 23:59 Beijing, even if we did not submit before 23:59 AOE and reviewed the results after 23:50 AOE. So is the time in the timeline AOE or Beijing? Or is it related to my region, I'm in China so it's Beijing time?
